electron/patch-desktop-filename.py

```diff
@@ -31,20 +34,26 @@
 import json
 import sys
 import os
+from pathlib import Path
 
 
 def main() -> None:
     asar_path = sys.argv[1]
-    extract_dir = tempfile.mktemp()
+    extract_dir = Path(tempfile.mktemp())
+    package_json_path = extract_dir / "package.json"
 
     asarPy.extract_asar(asar_path, extract_dir)
 
-    with open(os.path.join(extract_dir, "package.json"), "r", encoding="utf-8") as f:
+    with open(package_json_path, "r", encoding="utf-8") as f:
         package_json = json.load(f)
```

Comment:
For future: we should probably catch exceptions for opening files and parsing JSON, and exit early if they raise exceptions.

Reply:
Fixed in https://github.com/snapcrafters/patch-desktop-file-name/pull/1/commits/a4dfe9a798d6223ff7ac1a9afb6c268468f6adc3